src/com/cft/contactmerge/tests/CompareContactPartsTest.java

```diff
@@ -8,97 +8,232 @@
 
     @Test
     void doNamesMatch_NoDifferentLastName() {
-        Contact c1 = new Contact();
-        c1.setFirstName("John");
-        c1.setLastName("Doe");
+        assertEquals(AnswerType.no, CompareContactParts.doNamesMatch("Jonh", "Doe", "John", "Adams"));
```

Comment:
Maybe consider using assertThat(value, is(value))...

Reply:
Will do.